.github/workflows/build.yml

```diff
@@ -0,0 +1,67 @@
+# This workflow builds FTL
+# It is analogous to Circle CI workflow here:
+# ../../.circleci/config.yml
+name: Build and test
+
+on:
+  push:
+    branches:
+     - '*'
+  pull_request:
+    branches:
+     - '*'
+
+jobs:
+  build:
+
+    strategy:
+      matrix:
+        include:
+         - circle_job: armv4t
+           bin_name: pihole-FTL-armv4-linux-gnueabi
+         - circle_job: armv5te
+           bin_name: pihole-FTL-armv5-linux-gnueabi
+         - circle_job: armv6hf
+           bin_name:  pihole-FTL-armv6-linux-gnueabihf
+         - circle_job: armv7hf
+           bin_name: pihole-FTL-armv7-linux-gnueabihf
+         - circle_job: armv8a
+           bin_name: pihole-FTL-armv8-linux-gnueabihf
+         - circle_job: aarch64
+           bin_name: pihole-FTL-aarch64-linux-gnu
+         - circle_job: x86_64
+           bin_name: pihole-FTL-linux-x86_64
+         - circle_job: x86_64-musl
+           bin_name: pihole-FTL-musl-linux-x86_64
+         - circle_job: x86_32
+           bin_name: pihole-FTL-linux-x86_32
+
+    container: pihole/ftl-build:v1.8-${{ matrix.circle_job }}
+
+    runs-on: ubuntu-latest
+    continue-on-error: true
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: "Build"
+      run: |
+        # Extract branch name or ref from GITHUB_REF variable
+        CIRCLE_TAG=""
+        BRANCH="test"
+        bash .circleci/build-CI.sh "-DSTATIC=${STATIC}" "${BRANCH}" "${CIRCLE_TAG}" "${{ matrix.circle_job }}"
```

Comment:
@PromoFaux see here

Reply:
Is it the `BRANCH` or the `CIRCLE_TAG` variable? I'm playing with this offline, too on another branch - as I want to add the SFTP into it

Reply:
See here for comparison: https://github.com/pi-hole/FTL/blob/198e7c61362e07b22baa8bdb2bb57dd1c53be0fc/.circleci/config.yml#L10 and also check out the CircleCI docs concerning these variables.

Reply:
The only thing we need to do here is using the GHA counterparts of these magic CircleCI variables